hazelcast/src/main/java/com/hazelcast/client/impl/connection/tcp/TcpClientConnectionManager.java

```diff
@@ -534,6 +535,10 @@ private void shutdownWithExternalThread() {
 
     @Override
     public Collection<ClientConnection> getActiveConnections() {
+        return (Collection) activeConnections.values();
+    }
+
+    public Collection<TcpClientConnection> getActiveTcpClientConnections() {
```

Comment:
This second method is not necessary. The users of `getActiveTcpClientConnections` can use `getActiveConnections` instead. I have checked them. `ClientConnection` interface is enough for them. They don't need need implementation class.

Reply:
Fixed